doc/classes/TextureLayered.xml

```diff
@@ -1,10 +1,23 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TextureLayered" inherits="Texture" version="4.0">
 	<brief_description>
-		Base class for 3D texture types.
+		Base class for texture types which represent an array of [Image]s of the same size and format.
 	</brief_description>
 	<description>
-		Base class for [Texture2DArray], [Cubemap] and [CubemapArray]. Cannot be used directly, but contains all the functions necessary for accessing the derived resource types. Data is set on a per-layer basis. For [Texture2DArray]s, the layer specifies the array layer.
+		Base class for [Texture2DArray], [Cubemap] and [CubemapArray]. Cannot be used directly, but contains all the functions necessary for accessing the derived resource types.
+		Data is set on a per-layer basis. For [Texture2DArray]s, the layer specifies the array layer.
+		All images need to have the same width, height and number of mipmap levels.
+		A [TextureLayered] can be loaded with [code]method ResourceFormatLoaderTextureLayered.load[/code].		
+		To create a [TextureLayered] texture, an image file needs to be "sliced". Internally, Godot uses the thirdparty basis_universal library by Binomial LLC for e.g. creating and reading [Texture2DArray], [Cubemap] and [CubemapArray] files.
+		Their file extension is normally [code].basis[/code]. Godot adjust this to its needs:
+		[codeblock]
+		- the Texture2DArray becomes a ".tex2darr"
+		- the Cubemap becomes a ".cube"
+		- the CubemapArray becomes a ".cubearr" 
+		[/codeblock]
+		The [code]*.basis[/code] file remembers how to apply the slicing.
+		basis_universal can be accessed via the Godot Editor (during the image file import) and the basisu commandline tool (see file [code]thirdparty/basis_universal/basisu_tool.cpp[/code] regarding [code]-tex_type 2darray[/code]). With this library you combine several image files (e.g. png) to one *.basis file. This file will be read and sliced when you load it (see [code]basisu_backend.compute_slice_crcs[/code]).
+		Godot makes use of the loaded slices using the [code]method RenderingDevice.texture_create_shared_from_slice[/code] method using the [code]enum RenderingDevice.TextureSliceType[/code].
```

Comment:
Similarly, this one is on the line. It is helpful for advanced users who may want to implement their own stuff using the VisualServer API, but for regular users it is very likely to confuse them.

Reply:
same as above

Reply:
cut

NOTE: seems the basisu command line tool got deleted